src/sodaspark/scan.py

```diff
@@ -274,6 +278,7 @@ def execute(
     df: DataFrame,
     *,
     soda_server_client: SodaServerClient | None = None,
+    as_frame: bool | None = False,
```

Comment:
let's make this plural, since it is more accurate: `as_frames`

Reply:
Updated to `as_frames`